docs/_interface/peer-to-peer.md

```diff
@@ -122,21 +122,24 @@ It is often useful for developers to connect to private test networks rather tha
 
 ## Static nodes
 
-Geth also supports static nodes. Static nodes are specific peers that are always connected to. Geth reconnects to these peers automatically when it is restarted. Specific nodes are defined to be static nodes by saving their enode addresses to a json file which must be stored in `datadir/geth/static-nodes.json`. The content of `static-nodes.json` should be formatted as follows:
+Geth also supports static nodes. Static nodes are specific peers that are always connected to. Geth reconnects to these peers automatically when it is restarted. Specific nodes are defined to be static nodes by adding their enode addresses to a config file. The easiest way to create this config file is to run:
 
-```javascript
-[
-  "enode://f4642fa65af50cfdea8fa7414a5def7bb7991478b768e296f5e4a54e8b995de102e0ceae2e826f293c481b5325f89be6d207b003382e18a8ecba66fbaf6416c0@33.4.2.1:30303",
-  "enode://pubkey@ip:port"
-]
 ```
+geth --network id <networkid> --datadir <datadir> dumpconfig > datadir/config.toml
```

Comment:
So `--networkid` only sets the "network id" used in p2p communication. It doesn't configure which chain you're actually following, which I assume was the intention here. I'm not sure a good way to generically say "network name here" because to start a certain network, the flag is the network's name. E.g. `--sepolia`, `--goerli`, etc.

Reply:
ah yea, I see what you mean.
```suggestion
geth --datadir <datadir> dumpconfig > datadir/config.toml
```